gridstatus/spp.py

```diff
@@ -694,7 +694,7 @@ def _get_load_and_forecast(self, verbose=False):
     # 15mb file with five minute resolution
 
     def get_raw_interconnection_queue(self, verbose=False) -> BinaryIO:
-        url = "https://opsportal.spp.org/Studies/GenerateActiveCSV"
+        url = "https://opsportal.spp.org/Studies/GenerateSummaryCSV"
```

Comment:
I found another [url](https://opsportal.spp.org/Studies/GIRequests) on the SPP website that includes all withdrawn projects:

![image](https://github.com/kmax12/gridstatus/assets/17532695/5497d9cc-5349-4ec9-9f49-d7075b5653b1)

Reply:
Good find! Can we also update `get_interconnection_queue` so the "WITHDRAWN" status maps to `InterconnectionQueueStatus.WITHDRAWN.value`?

Reply:
Ah yes thank you! I just mapped "WITHDRAWN" to the `InterconnectionQueueStatus.WITHDRAWN.value` and added the withdrawn date. About a third of withdrawn projects do not have a withdrawn date. The vast majority of withdrawn projects missing a withdrawn date are from the early 2000s.